docs/installation.md

```diff
@@ -49,7 +57,7 @@ Zonemaster Engine test framework installed. Please see the
     $ zonemaster-cli example.com
 
 
```

Comment:
On line 47 it says "still as root". Instead it should be "su" or "sudo" (after it has been decided which one to use in all installation documents for FreeBSD).